api/src/test/java/org/openmrs/api/LocationServiceTest.java

```diff
@@ -228,7 +228,23 @@ public void getLocation_shouldReturnNullWhenNoLocationMatchGivenLocationId() {
 	public void getLocation_shouldReturnNullWhenNoLocationMatchGivenLocationName() {
 		Assert.assertNull(Context.getLocationService().getLocation("Princeton Plainsboro"));
 	}
-	
+
+	/**
+	 * @see LocationService#getLocation(String,String)
+	 */
+	@Test
+	public void getLocationWithAttribute_shouldReturnNullWhenNoLocationMatchGivenLocationId() {
+		Assert.assertNull(Context.getLocationService().getLocation("1337","locationId"));
```

Comment:
How can you ensure that there is no location with locationId as 1337? Better to prove that before testing your implementation.